src/Evolution/Systems/GeneralizedHarmonic/CharFields.hpp

```diff
@@ -0,0 +1,89 @@
+// Distributed under the MIT License.
```

Comment:
Why not put the char fields and speeds into Characteristics.?pp? Why the separate files CharFields.?pp here?

Reply:
Yep, thats the plan. Once #1071 is merged, I'll move code in CharFields.?pp to Characteristics.?pp. Till then it seemed cleaner to keep the files separate in case code in Characteristics.* underwent changes.